From: Luca Ceresoli <luca.ceresoli@bootlin.com>
Subject: scripts/spelling.txt: add "exsits" pattern and fix typo instances
Date: Thu, 26 Jan 2023 16:22:05 +0100

Fix typos and add the following to the scripts/spelling.txt:

  exsits||exists

Link: https://lkml.kernel.org/r/20230126152205.959277-1-luca.ceresoli@bootlin.com
Signed-off-by: Luca Ceresoli <luca.ceresoli@bootlin.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/drivers/infiniband/ulp/iser/iscsi_iser.c~scripts-spellingtxt-add-exsits-pattern-and-fix-typo-instances
+++ a/drivers/infiniband/ulp/iser/iscsi_iser.c
@@ -446,7 +446,7 @@ iscsi_iser_conn_create(struct iscsi_cls_
  * @is_leading:      indicate if this is the session leading connection (MCS)
  *
  * Return: zero on success, $error if iscsi_conn_bind fails and
- *         -EINVAL in case end-point doesn't exsits anymore or iser connection
+ *         -EINVAL in case end-point doesn't exists anymore or iser connection
  *         state is not UP (teardown already started).
  */
 static int iscsi_iser_conn_bind(struct iscsi_cls_session *cls_session,
--- a/net/openvswitch/flow_table.c~scripts-spellingtxt-add-exsits-pattern-and-fix-typo-instances
+++ a/net/openvswitch/flow_table.c
@@ -1012,7 +1012,7 @@ static int flow_mask_insert(struct flow_
 
 	mask = flow_mask_find(tbl, new);
 	if (!mask) {
-		/* Allocate a new mask if none exsits. */
+		/* Allocate a new mask if none exists. */
 		mask = mask_alloc();
 		if (!mask)
 			return -ENOMEM;
--- a/scripts/spelling.txt~scripts-spellingtxt-add-exsits-pattern-and-fix-typo-instances
+++ a/scripts/spelling.txt
@@ -630,6 +630,7 @@ exeuction||execution
 existance||existence
 existant||existent
 exixt||exist
+exsits||exists
 exlcude||exclude
 exlcusive||exclusive
 exlusive||exclusive
--- a/sound/soc/fsl/fsl-asoc-card.c~scripts-spellingtxt-add-exsits-pattern-and-fix-typo-instances
+++ a/sound/soc/fsl/fsl-asoc-card.c
@@ -811,7 +811,7 @@ static int fsl_asoc_card_probe(struct pl
 	priv->card.num_links = 1;
 
 	if (asrc_pdev) {
-		/* DPCM DAI Links only if ASRC exsits */
+		/* DPCM DAI Links only if ASRC exists */
 		priv->dai_link[1].cpus->of_node = asrc_np;
 		priv->dai_link[1].platforms->of_node = asrc_np;
 		priv->dai_link[2].codecs->dai_name = codec_dai_name;
_
